plugins/net-forward.yaml

```diff
@@ -0,0 +1,47 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+command: ./kubectl-net_forward
```

Comment:
this should be causing failure. unknown field.

Reply:
Fixed my misunderstanding of how sym-linking was working. Tested locally. Now there is a "net-forward" binary in v1.0.3